lib/features/children/add_member/pages/add_member_form_page.dart

```diff
@@ -6,6 +6,8 @@ import 'package:givt_app/features/children/add_member/widgets/add_member_form.da
 import 'package:givt_app/features/children/add_member/widgets/success_add_member_page.dart';
 import 'package:givt_app/features/children/add_member/widgets/vpc_page.dart';
 import 'package:givt_app/l10n/l10n.dart';
+import 'package:givt_app/shared/widgets/custom_green_elevated_button.dart';
```

Comment:
First comment is based on user testing: the checkbox looks a bit bigger then in the Figma file (both parts I think, but at least green background).
Also, it's not clear shall the animation be looped (perhaps it makes sense from explanatory perspective(in Figma it looks more clear with looped animation I think)?

Reply:
Yes you are right, it is bigger than the design. I will make it smaller. And no I don't think its looped. In the ticket it's a gif wich are always looped but it looks strange. But in the figma prototype its just a one time animation.
here:
https://www.figma.com/proto/dBOnDJTvfqLwJsK5yvXukj/Givt4Kids---Ongoing-Design?page-id=7420%3A5136&type=design&node-id=8091-18093&viewport=1393%2C819%2C0.29&t=lTLTb3st5FAnqULW-1&scaling=min-zoom&starting-point-node-id=7432%3A25820&mode=design

But looking at it agai, it needs a delay and to be bouncier.